backend/dal/Configuration/Views/PropertyConfiguration.cs

```diff
@@ -73,6 +73,8 @@ public virtual void Configure(EntityTypeBuilder<Entities.Views.Property> builder
                             AssessedDate = b.Evaluations.Any(f => f.Key == EvaluationKeys.Assessed) ? b.Evaluations.OrderByDescending(f => f.Date).FirstOrDefault(f => f.Key == EvaluationKeys.Assessed).Date : (DateTime?)null,
                             Appraised = b.Evaluations.Any(f => f.Key == EvaluationKeys.Appraised) ? b.Evaluations.OrderByDescending(f => f.Date).FirstOrDefault(f => f.Key == EvaluationKeys.Appraised).Value : 0,
                             AppraisedDate = b.Evaluations.Any(f => f.Key == EvaluationKeys.Appraised) ? b.Evaluations.OrderByDescending(f => f.Date).FirstOrDefault(f => f.Key == EvaluationKeys.Appraised).Date : (DateTime?)null,
+                            Improvements = b.Evaluations.Any(f => f.Key == EvaluationKeys.Improvements) ? b.Evaluations.OrderByDescending(f => f.Date).FirstOrDefault(f => f.Key == EvaluationKeys.Improvements).Value : 0,
```

Comment:
This section of the configuration is for unit-testing and the **In Memory DB**.  If we're not using the `ImprovementsDate`, I'd be hesitant to use it.  Additionally you will need to update the `View_Properties` to have the `Improvements` value, or your view will not work.  This will require a DB migration with a script to update and rollback the view.  I wouldn't include the `ImprovementsDate`, as it will be another additional join statement on the `View_Properties` that will slow it down even more.

Reply:
I ended up taking this out.

Reply:
@Fosol this was the flag that needs clearing that I referred to in scrum. it does appear that I resolved this by removing it.